deploy/csi-blobfuse-controller.yaml

```diff
@@ -20,11 +20,13 @@ spec:
       priorityClassName: system-cluster-critical
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.0.1
+          image: quay.io/k8scsi/csi-provisioner:v1.4.0
```

Comment:
I updated these images to use the `leases` lock, is that safe to upgrade them? @andyzhangx